cmd/cql/internal/create.go

```diff
@@ -104,6 +162,26 @@ func runCreate(cmd *Command, args []string) {
 		}
 	}
 
-	ConsoleLog.Infof("the newly created database is: %#v", dsn)
-	fmt.Println(dsn)
+	fmt.Printf("\nThe newly created database is: %#v\n", dsn)
+	storeOneDSN(dsn)
+	fmt.Printf("The connecting string beginning with 'covenant://' could used as a dsn for `cql console`\n or any command, and website like https://web.covenantsql.io\n")
```

Comment:
`covenant` -> `covenantsql`
`could used` -> `could be used`

Reply:
And don't understand what it means 'and website like https://web.covenantsql.io'